scripts/eval/eval-suite.py

```
@@ -0,0 +1,490 @@
+        "model": args.model,
+        "scope": args.scope,
+        "changed_files": len(changed_files),
+        "classification": {k: v for k, v in classified.items() if v},
+        "results": {},
+    }
+
+    any_failure = False
+
+    # 3a: Structural tests (ADR-023)
+    if classified["structural_test_targets"] and args.scope in ("prompts", "all"):
+        print(f"\n--- Structural Tests (ADR-023) ---", file=sys.stderr)
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)